src/scheduler/pbs_sched.c

```diff
@@ -568,7 +568,7 @@ server_command(char **jid)
 	}
 
 	if (set_nodelay(new_socket) == -1) {
-		sprintf(log_buffer, "cannot set nodelay on primary socket connection %d (errno=%d)\n", new_socket, errno);
+		snprintf(log_buffer, sizeof(log_buffer), "cannot set nodelay on primary socket connection %d (errno=%d)\n", new_socket, errno);
```

Comment:
In order to get errno in case of Windows OS you need to use WSAGetLastError. So wherever you are expecting/logging errno better to use #ifdef WIN32 for Windows Code. Please apply this comment wherever it is needed. 
Also please make sure that set_nodelay() function indeed works for Windows also. If the current definition does not work we need to modify it accordingly.